newdle/client/src/components/ParticipantTablev2.js

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
+import moment from 'moment';
+import {useSelector} from 'react-redux';
+import {Icon, Table} from 'semantic-ui-react';
+import {serializeDate, toMoment} from '../util/date';
+import {
+  getNewdleDuration,
+  getNewdleParticipants,
+  getNewdleTimezone,
+  getNewdleTimeslots,
+} from '../selectors';
+import UserAvatar from './UserAvatar';
+import styles from './creation/userSearch/UserSearch.module.scss';
```

Comment:
I'd copy the rule you are using into a stylesheet related to this component. all it contains is `border-radius: 50%;` so i don't think this is worth adding a dependency on the style module of a much more specific component here